src/popup/components/Modal.vue

```diff
@@ -8,6 +8,7 @@
       class="modal"
       :class="{
         'full-screen': fullScreen,
+        'full-width': fullWidth,
```

Comment:
Tsvetan told me that we won't add this new full width modals so this can be removed.